### DISCUSSION TITLE
Winlogbeats SSL Assistance Please

### DISCUSSION THREAD

**Discussion by MrDigitalDick** [Security-Onion-Solutions/securityonion#8429] [Unsupported Versions] [answered]
Hi, 

I have security onion 2.3, its all installed on one VM on VMWare. I have Net flow and Syslog from Meraki being piped in to it and that's working fine. Next I want to add winlogbeats on Windows computers, to which we have successfully setup and is also working. Now I would like to setup the SSL side of thing for it. I am following these instructions here:

[https://docs.securityonion.net/en/2.3/beats.html](url)

The part in bold, "Copy your certificates to the proper...." where do i get the certificates to copy? Any help gratefully received.

Using the Beats default port 5044 with encryption
Copy 0009_input_beats.conf to the local directory:

cp /opt/so/saltstack/default/salt/logstash/pipelines/config/so/0009_input_beats.conf /opt/so/saltstack/local/salt/logstash/pipelines/config/so/0009_input_beats.conf
**Copy your certificates to the proper directory on the manager. You will need a cert from the ca that you are signing the cert from, as well as the cert and key.**

cp myca.crt /opt/so/saltstack/local/salt/logstash/etc/certs/
cp mybeats.crt /opt/so/saltstack/local/salt/logstash/etc/certs/
cp mybeats.key /opt/so/saltstack/local/salt/logstash/etc/certs/
Next make your config look like the one below. Note that the paths are not the same due to docker.

Regards,
Rich

**Comment by dougburks** [comment, marked as answer]
You should be able to generate certificates using any standard CA (certificate authority).

**Reply by MrDigitalDick** [reply]
Im completely new to certificates, so not sure where id get a CA from. Would that be like a 3rd party provider or is it something I can install on say a server? Any linkns to some tutorials would be great and thanks again for your help Doug!

**Reply by dougburks** [reply]
There are lots of different CA options out there. Google can help you find them and their related documentation. Security Onion includes openssl and you can see some of the default certs in `/etc/pki/`.
